### ISSUE TITLE
ng-select input autocomplete contains dropdownId

### ISSUE THREAD

**Issue by Viale-Stephane** [ng-select/ng-select#2155]
**Describe the bug**
Hello, after running accessibility tool on the ng-select element (axe plugin) there's this issue "autocomplete attribute must be used correctly"

**To Reproduce** 
Steps to reproduce the behavior (if example is not provided):
1. Go to the demo page https://ng-select.github.io/ng-select#/data-sources
2. Click on trigger axe scan
4. See axe report with the previously mentioned issue.

**Expected behavior**
Should not have an accessibility issue

**Screenshots**
![image](https://user-images.githubusercontent.com/34646444/199939549-28a75148-fc73-4c62-8866-59b15fcf8a19.png)

**Desktop (please complete the following information):**
 - OS: W10
 - Browser: chrome
 - Version : 106.0.5249.119 (Official Build) (64-bit)

**Additional context**
What's happening is that you are putting by default the dropdownId inside the autocomplete attribute (except if the labelForId attribute is defined). https://github.com/ng-select/ng-select/blob/master/src/ng-select/lib/ng-select.component.ts#L775

From the WCAG21 (https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html): "The HTML autocomplete attribute only accepts a certain number of specific well-defined fixed values."
The list of accepted values can be found here: https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill

Is there a reason why you put the dropdownId inside the autocomplete attribute of the input ?
Thanks for your help !

**Comment by neilmenon** [comment]
Bump on this. This is an accessibility violation! Thanks!
